### ISSUE TITLE
Resources

### ISSUE THREAD

**Issue by martoio** [mkabobel/teacozy#1]
Make sure any resource you are referencing actually exists. The images are not part of the repository, so when I open it, I get loads of blank spaces and errors:

https://github.com/mkabobel/teacozy/blob/fd417c755d1e5beb1b7d05ef5a7ffe3a6e7b1f88/Documents/DesignWork/Projects/TeaCozy/index.html#L14

**Comment by mkabobel** [comment]
Yeah, I think I messed this up somehow when I was using git to push changes.
